src/utils/options.f90

```diff
@@ -51,6 +51,15 @@ module options
         ! where k_max is the maximum x or y wavenumber and zeta_char is
         ! a characteristic vorticity (see subroutine adapt of strat.f90).
         ! Note: nnu = 3 and prediss = 10 are recommended.
+
+        ! Prefactor type to use:
+        ! - vorch: characteristic vorticity
+        ! - roll-mean-gmax: rolling mean of gamma_max
+        character(len=16) :: pretype = 'vorch'
```

Comment:
To test the rolling mean, don't you need to choose roll-mean-gmax instead of vorch here?  And, should this be done at compile time rather than hard-coded?

Reply:
This is a job parameter. I use 'vorch' as default value to be backwards compatible.